wallet-new/src/Cardano/Wallet/API/V1/Handlers/Addresses.hs

```diff
@@ -7,19 +7,19 @@ import           Universum
 
 import           Servant
 
-import           Cardano.Wallet.WalletLayer.Types (ActiveWalletLayer,
-                     PassiveWalletLayer (..), walletPassiveLayer)
+import           Cardano.Wallet.WalletLayer.Types (PassiveWalletLayer (..))
 
 import           Cardano.Wallet.API.Request
 import           Cardano.Wallet.API.Response
 import qualified Cardano.Wallet.API.V1.Addresses as Addresses
 import           Cardano.Wallet.API.V1.Types
 
+
 import           Pos.Core.Mockable.Production (Production, runProduction)
 
-handlers :: ActiveWalletLayer Production -> ServerT Addresses.API Handler
+handlers :: PassiveWalletLayer Production -> ServerT Addresses.API Handler
 handlers w =  listAddresses
-         :<|> newAddress (walletPassiveLayer w)
```

Comment:
I have changed the type signature of these handlers to take only the `PassiveWalletLayer`, as they don't need anything else.